src/dex-ui/pages/CreateProposal/AddNewToken.tsx

```diff
@@ -1,36 +1,43 @@
 import { FormControl, HStack, Input, VStack } from "@chakra-ui/react";
+import { ChangeEvent } from "react";
 
-export function AddNewToken(props: any) {
+interface NewTokenProps {
+  title: string;
+  handleTitleChange: (event: ChangeEvent<HTMLInputElement>) => void;
+}
+
+export function AddNewToken(props: NewTokenProps) {
+  const { title, handleTitleChange } = props;
   return (
     <VStack alignItems="left" gap="10px">
       <FormControl>
-        <Input placeholder="Proposal Title" />
+        <Input value={title} onChange={handleTitleChange} variant="form-input" placeholder="Proposal Title" />
```

Comment:
Title is currently the only user entered field that is sent to the governor contract to create a proposal.